torchtune/utils/quantization.py

```diff
@@ -22,7 +22,19 @@
 
 
 if TORCH_VERSION_AFTER_2_3:
-    from torchao.quantization.quant_api import Int8DynActInt4WeightQuantizer
+    from torchao.quantization.quant_api import (
```

Comment:
@ebsmothers Can you remind me what our principles are around version guards? now that 2.4 is launched do we just claim we work with stable and remove such guards? or what's the downside of doing this?

Reply:
Yeah we can remove, our assumption is that users are at least on the latest stable version of PyTorch (so at this moment 2.4)